i18n/src/parser.js

```diff
@@ -33,7 +46,12 @@ function parsePattern(pattern) {
       positive = patternParts[0],
       negative = patternParts[1];
 
-  var positiveParts = positive.split(DECIMAL_SEP),
+  // The parsing logic below assumes that there will always be a DECIMAL_SEP in the pattern.
```

Comment:
Ok, I see. Can you please a short summary of this to the comment? I think this will be helpful.